src/effects/PitchEffect.js

```diff
@@ -8,32 +8,38 @@ var Tone = require('tone');
 
 function PitchEffect () {
     this.value = 0;
+    this.ratio = 1;
 
     this.tone = new Tone();
 }
 
 PitchEffect.prototype.set = function (val, players) {
     this.value = val;
+    this.ratio = this.getRatio(this.value);
     this.updatePlayers(players);
 };
 
 PitchEffect.prototype.changeBy = function (val, players) {
     this.set(this.value + val, players);
 };
 
-PitchEffect.prototype.getRatio = function () {
-    return this.tone.intervalToFrequencyRatio(this.value / 10);
+PitchEffect.prototype.getRatio = function (val) {
+    return this.tone.intervalToFrequencyRatio(val / 10);
+};
+
+PitchEffect.prototype.updatePlayer = function (player) {
+    player.setPlaybackRate(this.ratio);
 };
 
 PitchEffect.prototype.updatePlayers = function (players) {
     if (!players) return;
 
-    var ratio = this.getRatio();
-    for (var i=0; i<players.length; i++) {
-        players[i].setPlaybackRate(ratio);
+    for (var md5 in players) {
+        this.updatePlayer(players[md5]);
```

Comment:
This might pick up properties added to `Object.prototype` and other things you don't want. To avoid that, add this check:
```js
for (var md5 in players) {
    if (players.hasOwnProperty(md5)) {
        this.updatePlayer(players[md5]);
    }
}
```
I see a few other `for (var _ in _)` below that could use this check as well.